fract4d/c/fract4dc/functions.cpp

```diff
@@ -183,7 +183,7 @@ ffHandle * ff_fromcapsule(PyObject *pyff)
     ffHandle *ff = (ffHandle *)PyCapsule_GetPointer(pyff, OBTYPE_FFH);
     if (NULL == ff)
```

Comment:
This bit doesn't make a lot of sense. We only log the value if it is null? (I have no recollection of this code and don't know if it is being used)

I would follow same pattern as elsewhere with DEBUG_CREATION ifdef